apps/aehttp/src/aehttp_dispatch_int.erl

```diff
@@ -203,7 +203,7 @@ handle_request('PostNameUpdateTx', #{'NameUpdateTx' := NameUpdateTxObj}, _Contex
                             nonce     => Nonce,
                             name_hash => DecodedNameHash,
                             name_ttl  => NameTTL,
-                            pointers  => jsx:decode(Pointers),
+                            pointers  => jsx:decode(Pointers,[{labels, atom}]),
```

Comment:
Wasn't the atoms supposed to be removed from naming?

Reply:
I think git just backstabbed me. Thanks

Reply:
Rebase to master should fix this.